src/imputation/imputation_main.py

```diff
@@ -147,6 +149,13 @@ def run_imputation(
             f"{survey_year}_tmi_trim_count_qa_{tdate}_v{run_id}.csv"
         )
 
+        trim_qa_filename = filename_survey_prefixer(trim_qa_filename, survey_type)
+        full_imp_filename = filename_survey_prefixer(full_imp_filename, survey_type)
+        wrong_604_filename = filename_survey_prefixer(wrong_604_filename, survey_type)
+        links_filename = filename_survey_prefixer(links_filename, survey_type)
+        trimmed_counts_filename = filename_survey_prefixer(trimmed_counts_filename,
+                                                           survey_type)
+
```

Comment:
As discussed, instead of passing "survey_type" pass the config. You can use ctr + shift + f to do a global find and replace in VS Code. 
Then you can remove the lines of code that add the survey year, run_id and date and just pass the middle bit of the name to your function